<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LargeTitleLabel" name="LargeTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="IconWidget" name="IconWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../assets/res.qrc">
          <normaloff>:/Icons/materialIcons/icons8_python.svg</normaloff>:/Icons/materialIcons/icons8_python.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="StrongBodyLabel" name="strongBodyLabel">
          <property name="text">
           <string>Python.exe 路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CaptionLabel" name="CaptionLabel">
          <property name="text">
           <string>在这里选择 Python.exe 解释器的路径</string>
          </property>
          <property name="lightColor" stdset="0">
           <color>
            <red>96</red>
            <green>96</green>
            <blue>96</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="OutlinedPushButton" name="OutlinedPushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>自动选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="OutlinedPushButton" name="OutlinedPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选择路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="IconWidget" name="IconWidget_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../assets/res.qrc">
          <normaloff>:/Icons/materialIcons/icons8_unicast.svg</normaloff>:/Icons/materialIcons/icons8_unicast.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="StrongBodyLabel" name="strongBodyLabel_2">
          <property name="text">
           <string>Pip 源</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CaptionLabel" name="CaptionLabel_2">
          <property name="text">
           <string>点击这里可以重新为 pip 测速并更新 pip 源头,在您下载缓慢时启用</string>
          </property>
          <property name="lightColor" stdset="0">
           <color>
            <red>96</red>
            <green>96</green>
            <blue>96</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="OutlinedPushButton" name="OutlinedPushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>自动选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="IconWidget" name="IconWidget_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../assets/res.qrc">
          <normaloff>:/Icons/materialIcons/icons8_speedometer.svg</normaloff>:/Icons/materialIcons/icons8_speedometer.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="StrongBodyLabel" name="strongBodyLabel_3">
          <property name="text">
           <string>选项优化(建议开启)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CaptionLabel" name="CaptionLabel_3">
          <property name="text">
           <string>NuitkaGUI 默认开启了一些优化选项,您可以关闭他们获得完全的自定义体验</string>
          </property>
          <property name="lightColor" stdset="0">
           <color>
            <red>96</red>
            <green>96</green>
            <blue>96</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="SwitchButton" name="SwitchButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="onText">
         <string>开启</string>
        </property>
        <property name="offText">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="IconWidget" name="IconWidget_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../assets/res.qrc">
          <normaloff>:/Icons/materialIcons/icons8_new.svg</normaloff>:/Icons/materialIcons/icons8_new.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="StrongBodyLabel" name="strongBodyLabel_4">
          <property name="text">
           <string>初始化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CaptionLabel" name="CaptionLabel_4">
          <property name="text">
           <string>在这里可以清除所有的配置文件,将软件恢复到最初的设置</string>
          </property>
          <property name="lightColor" stdset="0">
           <color>
            <red>96</red>
            <green>96</green>
            <blue>96</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="OutlinedPushButton" name="OutlinedPushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>初始化</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OutlinedPushButton</class>
   <extends>QPushButton</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qmaterialwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>LargeTitleLabel</class>
   <extends>QLabel</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/res.qrc"/>
 </resources>
 <connections/>
</ui>
